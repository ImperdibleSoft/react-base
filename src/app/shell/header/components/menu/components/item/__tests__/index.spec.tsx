import * as React from 'react';
import { shallow } from 'enzyme';

import { Item } from '..';

describe('Item', () => {
  it('should render a valid external link', () => {
    const component = shallow(
      <Item theme={{ active: 'light' }} to="" isExternal>
        Test link
      </Item>
    );

    expect(component).toMatchSnapshot();
  });

  it('should render a valid local link', () => {
    const component = shallow(
      <Item theme={{ active: 'light' }} to="">
        Test link
      </Item>
    );

    expect(component).toMatchSnapshot();
  });
});
