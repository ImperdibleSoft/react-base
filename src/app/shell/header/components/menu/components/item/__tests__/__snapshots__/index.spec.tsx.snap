// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item should render a valid external link 1`] = `
<item__ExternalItem
  href=""
  rel="noopener noreferer"
  target="_blank"
>
  Test link
</item__ExternalItem>
`;

exports[`Item should render a valid local link 1`] = `
<item__LocalItem
  to=""
>
  Test link
</item__LocalItem>
`;
